README.md

```diff
@@ -11,6 +11,12 @@ As a prerequisite to build the parser, the following dependencies need to be ins
 
 **Note**: GNAT must be added to the PATH environment variable after installation: `export PATH=<GNAT-directory>/bin:$PATH`.
 
+The configuration of the development tools is managed in a separate repository and must be downloaded and set up once. The origin of the repository can be defined by setting the `RECORDFLUX_ORIGIN` variable.
```

Comment:
Do we need to mention the other variable (`ADACORE_ORIGIN`)?

Reply:
We could also consider not mentioning these variables at all in the README, as there should be no need to change them in the public repositories. What do you think?

Reply:
That's true. For the public repo it should just work and otherwise you have to know what you're doing. Let's remove it.

Reply:
Removed.